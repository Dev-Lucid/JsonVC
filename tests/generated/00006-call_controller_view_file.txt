{"title":"","description":"","keywords":"","test":"","author":"position set from controller view file in 00006","js":"","prepend":[],"append":[],"replace":[]}