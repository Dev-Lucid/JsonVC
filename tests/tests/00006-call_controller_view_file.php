<?
$controller = jvc::controller('00006');
$controller->do_test();
file_put_contents($output_path,json_encode($__jvc['response']));
?>