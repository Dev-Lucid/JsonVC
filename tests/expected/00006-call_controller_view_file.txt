{"title":"","description":"","keywords":"","author":"position set from controller view file in 00006","js":"","prepend":[],"append":[],"replace":[]}